checkout module assignment/Merve/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700&display=swap" rel="stylesheet">
+    <title>Checkout Form</title>
+</head>
+<body>
+    <div class="container__main--parent">
+        <div class="container__main--leftchild">
+            <nav class="nav__list">
+                <ul>
+                    <li class="nav__list--item">
+                        <span>1</span>
+                        <div>
+                            <p>STEP 1</p>
+                            <a href="#">YOUR INFO</a>
+                        </div>
+                    </li>
+                    <li class="nav__list--item">
+                        <span>2</span>
+                        <div>
+                            <p>STEP 2</p>
+                            <a href="#">SELECT PLAN</a>
+                        </div>
+                    </li>
+                    <li class="nav__list--item">
+                        <span>3</span>
+                        <div>
+                            <p>STEP 3</p>
+                            <a href="#">ADD-ONS</a>
+                        </div>
+                    </li>
+                    <li class="nav__list--item">
+                        <span>4</span>
+                        <div>
+                            <p>STEP 4</p>
+                            <a href="#">SUMMARY</a>
+                        </div>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div class="container__main--rightchild">
+            <div class="container__main--rightchild--header">
+                <h1>Personal Info</h1>
+                <p>Please provide your name, email address, and phone number.</p>
+            </div>
+            <div class="container__main--rightchild--form">
+                <form action="">
+                    <label for="name">Name</label>
+                    <input type="text" id="name" placeholder="e.g. Stephen King" required>
+                    <label for="email">Email Address</label>
+                    <input type="email" id="email" placeholder="e.g. stephenking@lorem.com" required>
+                    <label for="phone">Phone Number</label>
+                    <input type="tel" id="phone" name="phone" placeholder="e.g. +1 234 567 890" pattern="\+?\d{1,4}?[-.\s]?(\d{1,3}[-.\s]?){1,4}" required>
+                    <button class="button" type="submit">Next Step</button>
```

Comment:
In the design, the button is placed on the left side in the form. You can achieve it by placing the <button> element inside a <div> element, then make that <div> a flex-container in CSS and use `justify-content: flex-end;`